packages/react/src/ReactCurrentBatchConfig.js

```diff
@@ -8,16 +8,27 @@
  */
 
 import type {BatchConfigTransition} from 'react-reconciler/src/ReactFiberTracingMarkerComponent';
+import type {
+  EventPriority,
+  NoEventPriorityType,
+} from 'react-reconciler/src/ReactEventPriorities';
 
 type BatchConfig = {
   transition: BatchConfigTransition | null,
+  eventPriority: EventPriority,
 };
 /**
  * Keeps track of the current batch's configuration such as how long an update
  * should suspend for if it needs to.
  */
 const ReactCurrentBatchConfig: BatchConfig = {
   transition: null,
+  // We can't directly import from reconciler but we want to ensure
+  // this default value matches up with the expected event priority
+  // for NoEventPriority so instead we assert the type and then
+  // cast to avoid the opaque type restriction
+  // $FlowFixMe[incompatible-type]: NoEventPriority is valid but EventPriority is opaque
+  eventPriority: (0: NoEventPriorityType),
```

Comment:
Event priority is a very Fiber centric concept where as this store is intended to be more generic.

Originally `BatchConfigTransition.transition` was supposed to be just the options passed in to the public API but now it has become this very Fiber centric thing even importing the type from Fiber.

Ideally it would be more generic so that it can be implemented in another implementation too.

Reply:
I changed it to a boolean since we really only have a binary behavior